src/main/java/com/codecool/barbershop/barbershop/client/ClientService.java

```diff
@@ -35,38 +51,22 @@ public Client getClientById(long clientId) throws Exception {
 
 
     public List<ClientSearchAutocompleteReq> searchClientWithAutocomplete() {
-        List<ClientSearchAutocompleteReq> clientList = new ArrayList<>();
-        List<Client> allClients = getAllClients();
+        List<ClientSearchAutocompleteReq> clientSearchAutocompleteReqList = new ArrayList<>();
+        List<Client> allClients = clientRepository.findAll();
 
         for (Client client : allClients) {
-            ClientSearchAutocompleteReq clientSearchAutocompleteReq = new ClientSearchAutocompleteReq();
-            clientSearchAutocompleteReq.setId(client.getClientId());
-            clientSearchAutocompleteReq.setFirstName(client.getFirstName());
-            clientSearchAutocompleteReq.setPhoneNo(client.getPhoneNo());
-            clientSearchAutocompleteReq.setFirstNameAndPhone("Name: " + client.getFirstName() + " | Phone: " + client.getPhoneNo());
-
-            clientList.add(clientSearchAutocompleteReq);
+            ClientSearchAutocompleteReq autocompleteReq = new ClientSearchAutocompleteReq();
```

Comment:
Nice name simplifaction(s).  I understand that you don't expect any other autocompletes (on other entieties).